src/koopmans/ml/_ml_models.py

```diff
@@ -18,12 +21,32 @@ def predict(self, x_test: np.ndarray) -> np.ndarray:
     def fit(self, X_train: np.ndarray, Y_train: np.ndarray):
         ...
 
+    def todict(self):
+        dct = dict(self.__dict__)
+        return dct
+
+    @classmethod
+    def fromdict(cls, dct):
+        return cls(**dct)
+
+    def save_to_file(self, save_dir: Path) -> None:
+        from koopmans.io import write_kwf
+        with open(save_dir / f'model.kwf', 'w') as fd:
```

Comment:
Do we want to give this a more specific name? How about `self.__class__.__name__.lower()`?